introduction.md

```diff
@@ -5,3 +5,12 @@ I am cyberis.
 You should be afraid... very afraid
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?